src/main/java/core/basesyntax/service/ReportMakerServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class ReportMakerServiceImpl implements ReportMakerService {
+    private static final String COMMA = ",";
+    private static final String LINE_SEPARATOR = "\n";
+    private static final String FIRST_ROW = "fruit,quantity";
+
+    @Override
+    public String getReport() {
```

Comment:
I think it's better to pass Storage Map as input parameter here

Reply:
fixed